toolchain/check/testdata/class/dynamic_function_modifiers.carbon

```diff
@@ -0,0 +1,101 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/class/dynamic_function_modifiers.carbon
```

Comment:
Maybe "virtual_function_modifiers"? Although these are one way to get dynamic dispatch, https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/classes.md#virtual-methods calls these "virtual methods" and "virtual override keywords"

Reply:
Ah, sure - done.